stl/inc/flat_set

```diff
@@ -555,42 +561,55 @@ private:
     }
 
     template <class _Ty>
-        requires _Keylt_transparent || is_same_v<_Ty, _Kty>
-    size_type _Erase(_Ty&& _Val) {
-        const auto [_First, _Last] = equal_range(_Val);
+    size_type _Erase(const _Ty& _Val) {

```

Comment:
I believe it's safe to use `const auto&` here, as the `auto&&` in `erase` seems only to serve for constraint dispatching.